### DISCUSSION TITLE
Get last coordinates of content ?

### DISCUSSION THREAD

**Discussion by pnqphongct** [Hopding/pdf-lib#1291] [Q&A]
Hi guys,
I'm looking for an efficient way to insert the signature(text/image) to the existing pdf (without form fields, merge fields,..v.v)
Currently, i just tried to draw the signature to the last page without coordinate. it will be append to the end of the page.
```javascript
const pages = pdfDoc.getPages()
const lastPage = pages[pages.length - 1]
lastPage.drawText("hello", {size: 20 })
```
![image](https://user-images.githubusercontent.com/110152862/183063065-4b3146b8-d5a1-4636-a95a-2c0345fce7b5.png)

Is there any way to retrieve the coordinate(x, y) right after the content where i can put my signature in correct place ?

**Comment by adminconvives** [comment]
For an already written pdf I think you'd have to place it manually with moveTo( x, y ) for ex, but if you want the cursor position after some content you just added yourself you could get it with getX and getY [https://pdf-lib.js.org/docs/api/classes/pdfpage#getx]
